components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge/src/Supervisor/SupervisorServices.cs

```diff
@@ -339,8 +339,13 @@ private class TwinHost : IDisposable, IProcessControl, IModuleConfig {
                         throw new InvalidConfigurationException(
                             "Missing IOTEDGE_IOTHUBHOSTNAME variable in environment");
                     }
-                    cs = $"HostName={hostName};DeviceId={endpointId};SharedAccessKey={secret}";
                     var edgeName = Environment.GetEnvironmentVariable("IOTEDGE_GATEWAYHOSTNAME");
+                    if (string.IsNullOrEmpty(edgeName)) {
```

Comment:
isn't the if reversed here? edgeName != null etcetc

Reply:
indeed, that is fixed now